src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -220,7 +218,6 @@ MavlinkMissionManager::update_safepoint_count(unsigned count)
 		_count[MAV_MISSION_TYPE_RALLY] = count;
 
 	} else {
-		PX4_ERR("WPM: can't save mission state");
```

Comment:
send_statustext_critical() does not repeat the string, so the above was duplicate.